resources/views/actions/create.blade.php

```diff
@@ -29,11 +29,15 @@
                             @endforeach
                         </select>
                     </div>
-                    <div class="form-item -half">
+                    <div class="form-item -third">
+                        <label class="field-label">CallPower Campaign ID</label>
+                        <input type="text" name="callpower_campaign_id" class="text-field" placeholder="e.g. 4" value="{{ old('callpower_campaign_id') }}">
```

Comment:
It'd be nice to conditionally hide this field if we're not making a `phone-call` action, but that's a little bit more complicated to do outside of React-land. If you feel up for a little vanilla JS though, it'd be a nice UX win!